tests/unit/backend/utils/field-validation/attachment-validation.spec.ts

```diff
@@ -0,0 +1,174 @@
+import { ValidateFieldError } from 'src/app/modules/submission/submission.errors'
+import { ProcessedAttachmentResponse } from 'src/app/modules/submission/submission.types'
+import { validateField } from 'src/app/utils/field-validation/'
+import { IFieldSchema } from 'src/types'
+
+type MakeFieldOptions = {
+  required?: boolean
+}
+
+type MakeResponseOptions = {
+  answer?: string
+  filename?: string
+}
+
+describe('Attachment validation', () => {
+  const makeField = (
+    fieldId: string,
+    size: string,
+    options?: MakeFieldOptions,
+  ) => {
+    const attachment = {
+      _id: fieldId,
+      fieldType: 'attachment',
+      attachmentSize: size,
+      required: true,
+      ...options,
+    }
+    return (attachment as unknown) as IFieldSchema
+  }
+
+  const makeResponse = (
```

Comment:
possible to use `generateAttachmentResponse` or `generateNewAttachmentResponse` from `helpers/generate-form-data.ts`?